src/fielddef.ts

```diff
@@ -265,6 +265,11 @@ export function normalize(fieldDef: ChannelDef, channel: Channel) {
     if (!compatible) {
       log.warn(warning);
     }
+    if (fieldDef.type === 'ordinal') {
+      if ((<PositionFieldDef>fieldDef).axis.format) {
```

Comment:
The `<PositionFieldDef>` should not be necessary. Why does it not work otherwise?